Tasks/InstallSSHKeyV0/installsshkey-util.ts

```diff
@@ -179,7 +179,9 @@ export class SshToolRunner {
         let deleteKey: string = tl.getTaskVariable(postDeleteKeySetting);
         if (deleteKey) {
             tl.debug('Deleting Key: ' + deleteKey);
-            tl.execSync(path.join(external, 'ssh-add.exe'), ['-d', deleteKey]);
+            tl.execSync('ssh-add.exe', ['-d', deleteKey]);
```

Comment:
I think this line should be:
`tl.execSync(this.getExecutable('ssh-add'), ['-d', deleteKey]);`

I'm putting up my own code review for this (I'm also adding Unit Tests).  I'll try to wait for yours to finish before I complete so you don't have to suffer through conflicts.

Reply:
See #8283